legal-api/src/legal_api/resources/v2/configuration.py

```diff
@@ -35,3 +35,69 @@ def get_configurations():
             configuration.json for configuration in configurations
         ]
     }), HTTPStatus.OK
+
+
+@bp.route('', methods=['PUT'])
+@cross_origin(origin='*')
+@jwt.has_one_of_roles([UserRoles.staff])
+def update_configurations():
+    """Update the configurations."""
+    json_input = request.get_json()
+    if not json_input:
+        return {'message': 'Request body cannot be blank'}, HTTPStatus.BAD_REQUEST
+
+    configurations = json_input.get('configurations', [])
+    if not configurations:
+        return {'message': 'Configurations list cannot be empty'}, HTTPStatus.BAD_REQUEST
+
+    valid_names, msg = has_validate_names(configurations)
+    if not valid_names:
+        return ({'message': msg}), HTTPStatus.BAD_REQUEST
+
+    numeric_names = {'NUM_DISSOLUTIONS_ALLOWED', 'MAX_DISSOLUTIONS_ALLOWED'}
```

Comment:
I think we may want to update the model to have this there.  We may want to update some of Leo's code for the **validate_value** function to use these too.

Reply:
Updated

Reply:
I think it will be good to make [these lines](https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/models/configuration.py#L76-L78) enums or something similar and also update the line here if they will be used frequently. 🤔

Reply:
Removed this